OOP/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using OOP;
+
+
+RationalNumber r1 = new RationalNumber(45,20);
+RationalNumber r2 = new RationalNumber(5, 50);
+
+
+
+
+Console.WriteLine(r1.ToString());
+Console.WriteLine(r2.ToString());
```

Comment:
```suggestion
Console.WriteLine(r1);
Console.WriteLine(r2);
```
`.ToString` вызывать здесь нет необходимости. Консоль тама это сделает если передаваемый ей объект не является строкой.